readme.txt

```diff
@@ -40,11 +40,65 @@ https://youtu.be/_m6YzR0j8Fs
 * Can be disabled for any page using the post metadata sidebar.
 * Uses Gutenberg editor custom tool to search for Wikipedia articles and visualize Wikipedia Preview for readers.
 
-**How to use**
+**How to add preview links to your site**
 
-1. [Download and install](https://wordpress.org/support/article/managing-plugins/#finding-and-installing-plugins) the Wikipedia Preview plug-in for free.
-2. [Add a Preview link](https://wordpress.org/support/article/paragraph-block/#insert-hyperlinks) to your site’s content (i.e. a word or a phrase you want to provide context for).
-3. Wikipedia Preview will automatically turn the link into a preview of the relevant Wikipedia article.
+**Option 1 - How to add Wikipedia Preview links using the Classic Editor:**
+
+* Add a Wikipedia article link to your site’s content using the same process you use to add hyperlinks to your articles.
+
+**Option 2 - How to add Wikipedia Preview links using the Gutenberg/ Block Editor:**
+
+1. Highlight the text you want to link to a Wikipedia article.
+2. Select ‘Wikipedia Preview’ from the menu.
+3. You will see a list of suggested articles. Select the one you want to link to.
+4. Wikipedia Preview will automatically turn the link into a preview of the relevant Wikipedia article. You can easily edit or remove this.
+
+== Installation ==
```

Comment:
The title 'installation' is repeated again in line 63, maybe here we can switch to something like 'Setting up Wikipedia Preview'?

Reply:
Could you bring your comment to the task (https://phabricator.wikimedia.org/T324596) for Sudhanshu and Purity to consider?